package.json

```diff
@@ -20,9 +20,6 @@
     "test-dependencies": "node ./scripts/testDependencies.js",
     "typecheck": "flow check"
   },
-  "workspaces": [
-    "vscode-extension"
-  ],
```

Comment:
Using yarn workspaces wrecks havoc with the VSCode extension publishing tool. It fails to find the right dependencies, duplicates deps, etc. I'm not sure if there was benefit we were getting from this, but removing it makes the bundle/publishing of the extension much easier.